source/docs/Sensors/ultrasonic.rst

```diff
@@ -102,30 +102,51 @@ Programming the Ultrasonic Distance Sensor
             //Include the Ping Library
             #include "Ping_ros.h"
             
-            /**
-             * Constructor
-             * Ping's ros threads (publishers and services) will run asynchronously in the background
-             */
-             
-            ros::NodeHandle nh; //internal reference to the ROS node that the program will use to interact with the ROS system
-            VMXPi vmx(true, (uint8_t)50); //realtime bool and the update rate to use for the VMXPi AHRS/IMU interface, default is 50hz within a valid range of 4-200Hz
             
-            PingROS ping(&nh, &vmx, 8, 9);
-            ping.Ping(); //Sends an ultrasonic pulse for the ping object to read
+            double ping_cm;
             
-            //Create an accessor function
-            double getDistance()
+            void ping_cm_Cb(const std_msgs::Float32::ConstPtr& msg)
```

Comment:
This function has no comment block and will be very confusing. The function title itself is ??